packages/crane/src/components/menu/LanguagePicker.js

```diff
@@ -86,31 +85,34 @@ export default class LanguagePicker extends Component {
       backgroundColor,
       backgroundColorHover,
       current,
-      onClickHandler,
+      onTranslationClick,
       showAbove,
       translations,
     } = this.props;
+    debugger;
     return (
       <Wrapper>
-        <Current backgroundColor={backgroundColor}>
-          {languageList[current] ? languageList[current].nativeName : null}
-        </Current>
+        <Current backgroundColor={backgroundColor}>{current || null}</Current>
```

Comment:
Replace the backgroundColor props with theming variables? This comment applies to all colors.